test/integration/controller/jobset_controller_test.go

```diff
@@ -711,6 +715,392 @@ var _ = ginkgo.Describe("JobSet controller", func() {
 				},
 			},
 		}),
+		ginkgo.Entry("startupPolicy with InOrder; suspend should keep jobs suspended", &testCase{
+			makeJobSet: func(ns *corev1.Namespace) *testing.JobSetWrapper {
+				return testJobSet(ns).Suspend(true).
+					StartupPolicy(&jobset.StartupPolicy{
+						StartupPolicyOrder: jobset.InOrder,
+					})
+			},
+			updates: []*update{
+				{
+					checkJobSetState: func(js *jobset.JobSet) {
+						matchJobSetReplicatedStatus(js, []jobset.ReplicatedJobStatus{
+							{Name: "replicated-job-b",
+								Suspended: 3,
+							},
+							{
+								Name:      "replicated-job-a",
+								Suspended: 1,
+							},
+						})
+					},
+				},
+			},
+		}),
+		ginkgo.Entry("startupPolicy with AnyOrder; suspend should keep jobs suspended", &testCase{
+			makeJobSet: func(ns *corev1.Namespace) *testing.JobSetWrapper {
+				return testJobSet(ns).Suspend(true).
+					StartupPolicy(&jobset.StartupPolicy{
+						StartupPolicyOrder: jobset.AnyOrder,
+					})
+			},
+			updates: []*update{
+				{
+					checkJobSetState: func(js *jobset.JobSet) {
+						matchJobSetReplicatedStatus(js, []jobset.ReplicatedJobStatus{
+							{Name: "replicated-job-b",
+								Suspended: 3,
+							},
```

Comment:
A few integration test cases have the first ReplicatedJobStatus with this unconventional format:

```go
{Name: "replicated-job-b",
  Suspended: 3,
},
```

with the subsequent ones using the conventional format:

```go
{
  Name:      "replicated-job-a",
  Suspended: 1,
},
```

Can we fix this to use the conventional format consistently?

Reply:
done.